<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgressWindow</class>
 <widget class="QDialog" name="ProgressWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>208</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="lintGroupBox">
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>ProjectName</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="labelTimeElapsed">
          <property name="text">
           <string>Time elapsed:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="timeElapsed">
          <property name="text">
           <string>00:00:00</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelETA">
          <property name="text">
           <string>Estimated time remaining:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="eta">
          <property name="text">
           <string>00:00:00</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelFilesProcessed">
          <property name="text">
           <string>File:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="filesProcessed">
          <property name="text">
           <string>0/0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QProgressBar" name="lintProgressBar">
        <property name="value">
         <number>24</number>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
        <property name="format">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="lintCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
